test/data/hashing/ReferenceEventHashAlgorithm.json

```diff
@@ -0,0 +1,41 @@
+{
```

Comment:
I would rename these files in the following manner - `event-hash-reference-document.json`